src/package_io/parse_input.star

```diff
@@ -447,7 +448,7 @@ def enrich_mev_extra_params(parsed_arguments_dict, mev_prefix, mev_port, mev_typ
                 "el_client_image": parsed_arguments_dict["mev_params"][
                     "mev_builder_image"
                 ],
-                "cl_client_image": "sigp/lighthouse",
+                "cl_client_image": "ethpandaops/lighthouse:deneb-free-blobs-57edc0f",
```

Comment:
We have now introduced a flag for `mev_builder_cl_image` so you can also override this.